test/integration/goldens/compute/com/google/cloud/compute/v1/stub/HttpJsonRegionOperationsStub.java

```diff
@@ -85,9 +85,7 @@
                       .build())
               .setOperationSnapshotFactory(
                   (GetRegionOperationRequest request, Operation response) -> {
-                    StringBuilder opName = new StringBuilder(response.getId());
-                    opName.append(":").append(request.getProject());
-                    opName.append(":").append(request.getRegion());
+                    StringBuilder opName = new StringBuilder(response.getName());
```

Comment:
Sorry, I missed it during the previous review, can you  generate the line 92 a bit differently. Instead of 
`response.getStatus().equals(Status.DONE)`
do it in reverse instead?
`Status.DONE.equals(response.getStatus())`

This should give the same result but is safer, because the second variant will not produce null pointer exception if `response.getStatus()` is null (it is a general well known trick of doing comparisons in reverse)